File: TDDMicroExercises/PHP/TurnTicketDispenser/TicketDispenser.php
<?php

namespace TDDMicroExercises\PHP\TurnTicketDispenser;

class TicketDispenser
{

	public function getTurnTicket()
	{
		$newTurnNumber 	= TurnNumberSequence::getNextTurnNumber();
		$turnTicket 	= new TurnTicket($newTurnNumber);

		return $turnTicket;
	}
}

File: TDDMicroExercises/CSharp/TurnTicketDispenser/TicketDispenser.cs
namespace TDDMicroExercises.TurnTicketDispenser
{
    public class TicketDispenser
    {
        public TurnTicket GetTurnTicket()
        {
            int newTurnNumber = TurnNumberSequence.GetNextTurnNumber();
            var newTurnTicket = new TurnTicket(newTurnNumber);

            return newTurnTicket;
        }
    }
}
